# Time-varying Linear Gaussian controller
uint32 dX
uint32 dU
float64[] K_t  # Should be T x Du x Dx
float64[] k_t  # Should by T x Du
